koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -234,14 +242,21 @@ def process(self, parquet_base_filename, daily_data_frames):
                 cluster_topology = accessor.get_openshift_topology_for_multiple_providers(ocp_provider_uuids)
             # Get matching tags
             matched_tags = self.get_matched_tags(ocp_provider_uuids)
-            for i, daily_data_frame in enumerate(daily_data_frames):
-                openshift_filtered_data_frame = self.ocp_on_cloud_data_processor(
-                    daily_data_frame, cluster_topology, matched_tags
-                )
+            daily_data_frames = pd.concat(daily_data_frames, ignore_index=True)
+            openshift_filtered_data_frame = self.ocp_on_cloud_data_processor(
+                daily_data_frames, cluster_topology, matched_tags
+            )
 
-                if self.provider_type in (Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL):
-                    self.create_partitioned_ocp_on_cloud_parquet(
-                        openshift_filtered_data_frame, parquet_base_filename, i
-                    )
-                else:
-                    self.create_ocp_on_cloud_parquet(openshift_filtered_data_frame, parquet_base_filename, i)
+            if self.provider_type in (
+                Provider.PROVIDER_GCP,
+                Provider.PROVIDER_GCP_LOCAL,
+                Provider.PROVIDER_AWS,
+                Provider.PROVIDER_AWS_LOCAL,
+                Provider.PROVIDER_AZURE,
+                Provider.PROVIDER_AZURE_LOCAL,
+            ):
+                self.create_partitioned_ocp_on_cloud_parquet(
```

Comment:
Note this function already had the logic to handle AWS/Azure files correctly splitting them into daily files (Sure the naming was wrong, till now) but we never passed them into this for some reason.

Reply:
Following up on this, the create partitioned OCP on Cloud parquet function was implemented because we partition GCP by `day` as well so the files needed to contain the specific records pertaining to the usage start date. What do we gain here by sending AWS/Azure through this flow? Doesn't this just create more individual parquet files for trino to have to open later?

Reply:
So the gain here, for AWS/Azure at least would be when deleting the parquet files for our relevant daily processing window we delete the correct corresponding files and data. The partitioned processing within this function is breaking those files out based on their usage dates. :slightly_smiling_face: So this means the files can be named and populated correctly for our deletion on consecutive daily runs.